tests/unit/types/arrays/test_memmap.py

```diff
@@ -173,3 +173,19 @@ def test_convert_dm_to_dam(tmpdir, mocker):
     mock.assert_called()
     assert len(da) == 0
     assert len(dam) == 100
+
+
+@pytest.mark.parametrize('embed_dim', [10, 10000])
+def test_extend_and_get_attribute(tmpdir, embed_dim):
+    dam = DocumentArrayMemmap(tmpdir)
+    docs = list(random_docs(100, start_id=0, embed_dim=embed_dim))
+    dam.extend(docs)
+
+    dam2 = DocumentArrayMemmap(tmpdir)
+    x = dam2.get_attributes('embedding')
+    assert len(dam2) == 100
+
+    docs = list(random_docs(100, start_id=100, embed_dim=embed_dim))
+    dam2.extend(docs)
+    x = dam2.get_attributes('embedding')
+    assert len(dam2) == 200
```

Comment:
```suggestion
    dam2 = DocumentArrayMemmap(tmpdir)
    x = dam2.get_attributes('embedding')
    assert len(dam2) == 100
    assert len(x) == 100
    assert x[0].shape == (embed_dim,)

    docs = list(random_docs(100, start_id=100, embed_dim=embed_dim))
    dam2.extend(docs)
    x = dam2.get_attributes('embedding')
    assert len(dam2) == 200
    assert len(x) == 200
    assert x[0].shape == (embed_dim,)
```

Reply:
Addressed, please see [e5dea0f](https://github.com/jina-ai/jina/pull/2956/commits/e5dea0f45fbd88593f8ae29405a8c43085a99996) for the change, thanks.